### PR TITLE
Add StashCache to VO template (SOFTWARE-3518)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: template-virtual-organization.yaml [hunk 1]
```diff
@@ -85,3 +85,16 @@ SupportURL: http://www.usatlas.bnl.gov/index.shtml
...
+### DataFederations (optional) is information about data federations that the VO is part of
+### This template is for the StashCache data federation; for instructions,
+### see the documentation at https://opensciencegrid.org/docs/data/stashcache/vo-data/
+# DataFederations:
+#   StashCache:
+#     Namespaces:
+#       /<VONAME>/PUBLIC:
+#         - ANY
+#     AllowedCaches:
+#       - ANY
```
Comment (brianhlin) [inline template-virtual-organization.yaml]: Do we want this to be the default suggestion? I'm inclined to say that we should just have:

```
- <CACHE_RESOURCE>
- ...
```

File: template-virtual-organization.yaml [hunk 2]
```diff
@@ -85,3 +85,16 @@ SupportURL: http://www.usatlas.bnl.gov/index.shtml
...
+#     AllowedOrigins:
+#       - <ORIGIN_RESOURCE>
```
Comment (brianhlin) [inline template-virtual-organization.yaml]: Since there can be many, should we add `- ...`?

Comment (matyasselmeci) [reply template-virtual-organization.yaml]: Done

File: template-virtual-organization.yaml (hunk 1 above)
Comment (matyasselmeci) [reply template-virtual-organization.yaml]: Done

**Pull request merged**